settings/UL/settings_embeddingselection_2017UL.yaml

```diff
@@ -79,11 +102,25 @@ ID_pt_eta_bins:
   ratio_y_range: [0.8, 1.2]
 
 EmbID_pt_bins_inc_eta:
-  # Settings for histogram creation
   name: "EmbID_pt_bins_inc_eta"
   var: "m_vis(100,50,150)"
-  tag: "trg_IsoMu27_tag && fsr_photon_veto_tag < 0.5 && fsr_photon_veto_probe < 0.5 && pt_tag > 28 && iso_tag < 0.15 && dz_tag < 0.2 && dxy_tag < 0.045 && id_medium_tag"
-  probe: "is_global_probe && id_loose_probe"
+  tag:  (
+          (iso_tag < 0.15) &&
+          ((fsr_photon_veto_tag < 0.5) && (fsr_photon_veto_probe < 0.5)) &&
+          (id_medium_tag) &&
+          (
+            (trg_IsoMu27__no_filterbit_tag) ||
+            (trg_IsoMu24__no_filterbit_tag)
+          ) &&
+          (pt_tag > 28.0) &&
+          ((q_1 + q_2) == 0) &&
+          (abs(dz_tag) < 0.2) &&
+          (abs(dxy_tag) < 0.045)
+        )
+  probe:  (
+            (is_global_probe) && 
+            (id_loose_probe)
```

Comment:
here `loose` is correct - please double-check for all eras

Reply:
See comment from settings_embeddingselection_2016preVFPUL.yaml